task06/server.ts

```diff
@@ -0,0 +1,17 @@
+import express from 'express';
+import productRouter from './routes/product.route';
+import cartRouter from './routes/cart.route';
+import { handleError } from './controller/error.controller';
+
+const PORT = 8000;
+const app = express();
+
+app.use(express.json());
+app.use('/api/products', productRouter);
+app.use('/api/profile/cart', cartRouter);
```

Comment:
Basically, you don't need to specify `/api` everywhere, you can make it global using middleware approach.

Reply:
fixed